dist/src/main/resources/api/backup-management-api.yaml

```
@@ -109,6 +110,77 @@ paths:
           description: Request from gateway to broker timed out
           $ref: '#/components/responses/Error'
 
+  # Section on historic backups
+  /backup-history:
+    post:
+      summary: Takes a historic backup
```

:wrench: 
```suggestion
      summary: Takes a backup of history data
```

To keep it consistent with the runtime one and to not imply the backup is of historic value ;) 
Same for the other endpoints.